changelogs/fragments/junos-ci-fix-d533a54d99180980.yaml

```diff
@@ -0,0 +1,2 @@
+bugfixes:
+  - netconf - Fix Zuul CI integration
```

Comment:
changelogs are for the end users.  So they should reflect what was fixed for them.  Tracebacks?  Idempotency?  Behaviour that didn't match the documentation?

Changes to tests don't need to be changeloged but the change to lib/ansible/plugins/netconf/__init__.py does.